src/wp-includes/link-template.php

```diff
@@ -1597,25 +1597,39 @@ function get_delete_post_link( $post = 0, $deprecated = '', $force_delete = fals
  * @since 2.3.0
  *
  * @param int|WP_Comment $comment_id Optional. Comment ID or WP_Comment object.
+ * @param string         $context    Optional. Context in which the URL should be used. Either 'display',
+ *                                   to include HTML entities, or 'display'. Default 'display'.
```

Comment:
There's something wrong here. It mentions the value `display` twice. The other documented value should be `url`, similar to how WordPress core already uses it for `get_edit_post_link()`'s `$context` parameter.